docs/mine/lotus/split-markets-miners.md

```diff
@@ -78,21 +78,26 @@ lotus-miner backup ~/lotus-backup-location/backupfile
 You need to generate a `config.toml` for the markets node, and have them ready for the next step. For more information see [configuration usage page](https://docs.filecoin.io/get-started/lotus/configuration-and-advanced-usage/) and the [custom storage layout page](https://docs.filecoin.io/mine/lotus/custom-storage-layout/).
 
 If you intend to run the `mining/sealing/proving` node on the same machine as the `markets` node, make sure that their listeners do not clash:
-```
+
+```toml
 [API]
   ListenAddress = "/ip4/127.0.0.1/tcp/8787/http"
   RemoteListenAddress = "127.0.0.1:8787"
 ```
 
-By default the `lotus-miner` node listens to port 2345, so in the example configuration above, we change the configuration for the `markets` node to listen to `127.0.0.1:8787`.
+By default the `lotus-miner` API server listens to port 2345, so in the example configuration above, we change the configuration for the `markets` node to listen to `127.0.0.1:8787`.
 
 In the example commands below, we have placed the `config.toml` in the `~/.lotusmarket` directory.
 
+Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it needs to be publicly accessible.
+
+The `[Libp2p]` section on the `mining/sealing/proving` node is no longer needed given that this node won't be running a Libp2p node.
```

Comment:
Suggest changing this to read
`node can be removed because this node will no longer be running a Libp2p node`